frontend/src/pages/CourseCreator/CourseCreatorContainer.tsx

```diff
@@ -0,0 +1,70 @@
+import { BaseSyntheticEvent, FC } from 'react';
+import { useLocation } from 'react-router-dom';
+import { useFormik, FormikProvider } from 'formik';
+
+import { useGetSkills } from 'api/skills';
+import { PATHS } from 'constants/routes';
+import { INITIAL_VALUES, RADIX_PARAMETER } from 'constants/courseEditor';
+import { courseEditorValidationSchema } from 'validations/schemas';
+import { formatFieldValue } from 'pages/CourseEditor/utils';
+
+import CourseCreator from './CourseCreateor';
+
+const CourseCreatorContainer: FC = () => {
+  const formik = useFormik({
+    initialValues: INITIAL_VALUES,
+    onSubmit: (): void => {},
```

Comment:
here we should also implement API request for course creating and add the function for submit here